src/glyphs.js

```diff
@@ -57,6 +64,28 @@ const processCacheRequest = req => {
   return processed;
 };
 
+const processMessageRequest = req => {
+  const processed = {
+    errors: [],
+    parsed: {}
+  };
+
+  if (!req.body.message || !req.body.prob) {
+    processed.errors.push('No data provided!');
+  }
+
+  try {
+    processed.parsed = JSON.parse(req.body.message);
```

Comment:
No need to prefix with `parsed`, as `processed` implies it.  I'd name these `processed.message` and `processed.probability`